src/components/MainActionButton.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Definitely want to make sure we take this out at some point. Don't forget you can run `npm run lint:fix` if you are having issues with linting